src/locales/en.json

```diff
@@ -152,7 +152,14 @@
     "allow": "Allow",
     "why": "Why do I need to do this?",
     "if_denied": "If you deny camera access, you won't be able to take pictures and complete the verification process.",
-    "enable_webcam": "Enable webcam"
+    "enable_webcam": "Enable webcam",
```

Comment:
We need to include Spanish translations for this screen

Reply:
Thanks @stefaniacardenas, just added those in